NEWS

```diff
@@ -24,13 +24,29 @@ Features:
 - Updated support for Mellanox DevX API
 - Added multiple UCT/TCP transport performance optimizations
 - Optimized memcpy() for Intel platforms
+- Adding protection from non-UCX socket based app connections
```

Comment:
maybe added to be consistent with other points?

Reply:
valid point. I will fix